js/app/modules/card/list.js

```diff
@@ -45,10 +46,14 @@ const List = new Module.Class({
         this.set_thumbnail_frame_from_model(this._thumbnail_frame);
         this.update_card_sizing_classes(Card.MinSize.A, Card.MinSize.D);
         this._synopsis_label.visible = this.show_synopsis;
-
+        this._title_label.vexpand = !this.show_synopsis;
         Utils.set_hand_cursor_on_widget(this);
     },
 
+    mark_read: function (is_read) {
```

Comment:
This would change given the refactors we discussed in the hangout today, but otherwise this commit looks OK to merge.